### DISCUSSION TITLE
Is there a way to add a post to a certain past date?

### DISCUSSION THREAD

**Discussion by pedrojvaz** [usememos/memos#4100] [Q&A] [answered]
As the title says. I want to be able to quickly add a post/entry to a day in the past, but they are all grayed out

Thanks

**Comment by RoccoSmit** [comment]
When you edit your note you can click on the date to change it. 
I believe you will need to be using v0.23 (latest docker image) for this to work.

**Reply by pedrojvaz** [reply]
Oh, I see. It's not as practical as clicking in the calendar though. Isn't there a way to start a new post and type the date, or something similar?

**Reply by RoccoSmit** [reply, marked as answer]
Not at this time

**Reply by bradyloveland** [reply]
I'm following up to this question as I've just started using the memos app.  Its great and thank you all for developing it!  I am currently working on migrating historical notes from DayOne by copy-pasting, but having to save the date after the memo is saved isn't as bad as having to click through the left arrow through months and months to get back to entries from 10+ years ago.  Is it possible to look into a different date picker for the date/time popup that would allow you to either manually enter the date, or have more control over quickly jumping to a year, day, month?  See attached for some examples of other date pickers.
<img width="473" alt="datepicker-b5cfaa0703a4fc34709e76cf7cf3383c" src="https://github.com/user-attachments/assets/434653a2-2a83-4b21-9f42-5abe0b5409ca" />
![Datepicker](https://github.com/user-attachments/assets/da2898fb-fadd-4d72-b263-79937cf81b8b)

**Reply by RoccoSmit** [reply]
Feature merged and will be in the next version of memos. See [here](https://github.com/usememos/memos/pull/4286)

**Reply by bradyloveland** [reply]
> Feature merged and will be in the next version of memos. See [here](https://github.com/usememos/memos/pull/4286)

Amazing, thanks so much! I’m eagerly awaiting the release 😀

**Reply by RoccoSmit** [reply]
v0.23.1 is out and contains the date selection dropdowns

**Comment by posta246** [comment]
Hi, I'm looking around to solve the same quest.
It seems the (amazing) API of memo do not have a created_date field that I can use to pass the past date of my notes, right? 
It is not clear because the field is in the request but read-only, and there is not in the example request, just in the response.
Other way I'm looking for is to insert data into the db directly but it seems more difficult, considering the hashtags etc.
Maybe the created_at param can be add in the request of the create memo method and saved?
Pleeeeeeeeeeease!

**Reply by RoccoSmit** [reply]
The `createTime` property is on the update memo endpoint. See UpdateMemo here https://memos.apidocumentation.com

**Reply by posta246** [reply]
Great Scott, have not considered it could be obtained using a different API method. Thanks a lot, I will play with it.

**Reply by posta246** [reply]
Hi, it seems it is not easy for me to understand how to build the json payloads of create a new memo and then modifyng the creation date (so the added memos appear as they where published in the past). I have collection of ("memo content", "memo date").
Might you give a little and further help?
